docs/guides/nwaku/run-docker.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Run Nwaku in a Docker Container
+---
+
+This guide provides detailed steps to build and run a `nwaku` node in a Docker container. If you prefer a pre-configured setup that includes a monitoring dashboard, see the [Run Nwaku with Docker Compose](/guides/nwaku/run-docker-compose) guide.
+
+## Prerequisites
+
+Ensure [Docker](https://www.docker.com/) is installed on your system using the appropriate instructions provided in the [Docker documentation](https://docs.docker.com/engine/install/). For example, you can use Docker's convenience script for installation:
+
+```bash
+curl -fsSL https://get.docker.com -o get-docker.sh
```

Comment:
Does this work on both MAc and Linux?

Reply:
I tried it out, didn't work on Mac. I'll remove that line